subprojects/kotlinReleaseCoroutinesTest/kotlinReleaseCoroutinesTest.gradle

```diff
@@ -0,0 +1,26 @@
+buildscript {
+    repositories {
+        mavenCentral()
+        maven { url 'https://dl.bintray.com/kotlin/kotlin-eap' }
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.0-rc-57"
+    }
+}
+
+apply plugin: "org.jetbrains.kotlin.jvm"
+
+description = "Kotlin tests for Mockito."
+
+repositories {
+    mavenCentral()
+    maven { url 'https://dl.bintray.com/kotlin/kotlin-eap' }
+}
+
+dependencies {
+    testCompile project(":")
+    testCompile libraries.junit4
+
+    testCompile  "org.jetbrains.kotlin:kotlin-stdlib:1.3.0-rc-57"
+    testCompile 'org.jetbrains.kotlinx:kotlinx-coroutines-core:0.26.1-eap13'
```

Comment:
We are using `dependencies.gradle` to define our dependencies (https://github.com/mockito/mockito/blob/release/2.x/gradle/dependencies.gradle). Could you use the same format as https://github.com/mockito/mockito/blob/release/2.x/subprojects/kotlinTest/kotlinTest.gradle and add the extra dependency in `dependencies.gradle`?

Reply:
Yes of course, I can. 
I haven't done that just to avoid introducing the version used only within a single test module

Reply:
Could you resolve this comment? Then I think this is ready for a merge.

Reply:
Done